src/rars/venus/editors/jeditsyntax/tokenmarker/RISCVTokenMarker.java

```diff
@@ -195,6 +196,33 @@ else if (c == '\'') {
         return token;
     }
 
+    private void findLabelKeywords() {
+        labelKeywords = new KeywordMap(false);
+        if (Globals.getGui().getMainPane().getEditPane() != null) {
+            char[] array = Globals.getGui().getMainPane().getEditPane().getSource().toCharArray();
+            int lastNewLine = 0;
+            int currentIndex = 0;
+
+            for (char c : array) {
+                if (c == '\n')
+                    lastNewLine = currentIndex;
+                if (c == ':') {
```

Comment:
There are already 2 tokenizers (`rars.assembler.Tokenizer` and `rars.venus.editors.jeditsyntax.tokenmarker`) ; there should be a way to reuse one of them or generalize it instead of parsing the label definition a 3rd way. 

Ideally, there should be only one tokenizers, but that's another battle :)